manifests/init.pp

```diff
@@ -80,11 +88,13 @@
             onlyif   => "! grep '^exclude=' /etc/yum.conf",
             user     => root,
             provider => shell,
-            require  => Exec['Ensure Service']
+            require  => Exec['Ensure Service'],
         }->
         exec { 'Yum Exclude gitlab-ci-multi-runner':
-            command  => "sed -i 's/^exclude=.*$/& gitlab-ci-multi-runner/' /etc/yum.conf",
-            onlyif   => "! grep '^exclude=.*gitlab-ci-multi-runner' /etc/yum.conf",
+            command  => "sed -i 's/^exclude=.*$/& gitlab-ci-multi-runner/' \
```

Comment:
To be under the 80 chars line length, as reported by default by puppet lint. I usually do it when possible.